### PR TITLE
 Automate application delivery with Azion Edge Orchestrator

### PR DESCRIPTION
**Pull request by gabriel-azion**

### CODE REVIEW

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/architectures/orch/application-deliver-operation.mdx [hunk 1]
```diff
@@ -0,0 +1,141 @@
+---
+title: Automate application delivery with Azion Edge Orchestrator
+description: >-
+  Learn how to streamline application delivery and infrastructure management using Azion Edge Orchestrator. Enhance performance with efficient edge node management and automated operations.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/orch/application-deliver-operation.mdx:4]:
```suggestion
  Streamline application delivery and infrastructure management with Azion Edge Orchestrator, enhancing performance with efficient management and automated operations.
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/architectures/orch/application-deliver-operation.mdx:4]: to be around 160 chars

File: src/content/docs/en/pages/architectures/orch/application-deliver-operation.mdx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+meta_tags: >-
+  Azion Edge Platform, Azion Edge Orchestrator, application delivery automation, edge node management, infrastructure automation, Azion CLI, performance optimization, edge computing
+namespace: docs_arch_infra_auto_apps_delivery
+---
+
+[Azion Edge Orchestrator](/en/documentation/products/deploy/edge-orchestrator/) enables frequent and reliable application releases through orchestration. This documentation covers the architecture and steps for automated application delivery using Azion Edge Orchestrator, ensuring optimal performance and streamlined operations.
+
+---
+
+## Architecture
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/orch/application-deliver-operation.mdx:15]:
```suggestion
## [use case name] Architecture Diagram
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/architectures/orch/application-deliver-operation.mdx:15]: to align with template

File: src/content/docs/en/pages/architectures/orch/application-deliver-operation.mdx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+:::note
+This flow assumes a service has been created for saving an HTML page containing a list of products. For further information, check the [Implementation](#implementation) section.
+:::
+
+![Orchestrator architecture diagram](/assets/docs/images/uploads/infrastructure-arch.png)
+
+### Dataflow
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/orch/application-deliver-operation.mdx:23]:
```suggestion
### [use case name] Dataflow
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/architectures/orch/application-deliver-operation.mdx:23]: same

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/architectures/orch/application-deliver-operation.mdx [hunk 4]
```diff
@@ -12,15 +12,15 @@ namespace: docs_arch_infra_auto_apps_delivery
 
 ---
 
-## Architecture
+## [use case name] Architecture Diagram
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/orch/application-deliver-operation.mdx:15]:
```suggestion
## Application Delivery and Operations Architecture Diagram
```

File: src/content/docs/en/pages/architectures/orch/application-deliver-operation.mdx [hunk 5]
```diff
@@ -12,15 +12,15 @@ namespace: docs_arch_infra_auto_apps_delivery
...
 
 :::note
 This flow assumes a service has been created for saving an HTML page containing a list of products. For further information, check the [Implementation](#implementation) section.
 :::
 
 ![Orchestrator architecture diagram](/assets/docs/images/uploads/infrastructure-arch.png)
 
-### Dataflow
+### [use case name] Dataflow
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/orch/application-deliver-operation.mdx:23]:
```suggestion
### Application Delivery and Operations Dataflow
```

**Pull request merged**